src/CSFeeOracle.sol

```diff
@@ -38,14 +39,18 @@ contract CSFeeOracle is
     /// performance over the network computed by the off-chain oracle.
     uint256 internal _avgPerfLeewayBP;
 
+    ICSStrikes public strikes;
```

Comment:
Re-use the slot above. I forgot to mention it as deprecated in my PR.

Reply:
We don't need to nullify it in `finalizeUpgradeV2` explicitly before assigning address, am I right?

Reply:
Yes